### DISCUSSION TITLE
Making map harder by using Mod Customization tool incrases Performance Points.

### DISCUSSION THREAD

**Discussion by PolskiKapsel** [ppy/osu#21161] [Q&A]
Hi there, 
if You try to make make harder via mods like Difficulty Adjust and Double Time it actually increases your Performance Points. Logically there's nothing wrong with that, but still it's kinda weird to see the score being chopped off but the PP is still higher adequately to how harder we made the map. It surely would be interesting to see how creative players farming PP get, but at the same time I have a feeling that it could be highly abused. 

Getting to the point now, 
Is it intentional or just Peppy didn't bother to make new calculations yet? 

![PP change](https://user-images.githubusercontent.com/102519385/200398304-ebbf17df-8494-4fb4-a5d5-5131e8b1a7bc.png)

**Comment by ghost** [comment]
~~#19222~~

~~you can see correct values when viewing more replay information though (click on the score)~~
misread the discussion nvm

**Comment by cdwcgt** [comment]
the pp here mostly from acc.
<details><summary>specific value</summary>

![image](https://user-images.githubusercontent.com/34775378/200484260-19d5838f-12a8-486f-b41f-c118313c2cdc.png)
</details>

this is probably because long drain time and get SS when OD10


<details>
<summary>OD 8.0</summary>

![image](https://user-images.githubusercontent.com/34775378/200484431-2ed325d5-3da6-47ce-b613-b1aa1399fce2.png)
</details>
I don't think it's a questionable calculation as it's really worth the pp to have such a high level of accuracy in the case of the AR10OD10

**Reply by PolskiKapsel** [reply]
I guess You are right. I was just shocked to see over 300 PP for a 4* map. Right now I'm hoping that Peppy wont make using Customisation Tool unranked.

**Reply by Rekunan** [reply]
It's also that after 2x AR10OD10 becomes AR11.5OD11.625 (or at least around these values if not exact)

@PolskiKapsel Every mod, except for auto, is and will stay ranked.
